3.10/aql/graphs-traversals.md

```diff
@@ -134,8 +134,9 @@ FOR vertex[, edge[, path]]
     case when a nested traversal is fed with several tens of thousands of start
     vertices, which can then be distributed randomly to worker threads for parallel
     execution.
-
-    {% include hint-ee.md feature="Traversal parallelization" plural=false %}
+    % include hint-ee.md feature="Traversal parallelization" plural=false %}
+  - **maxProjections** (number, *optional*): Specifies the number of graph
```

Comment:
```suggestion
  - **maxProjections** (number, *optional*): Specifies the number of document
```

We are projecting on the data, not the "graph" itself.
This could be a bit misleading but I am not really sure myself.